cpplib/src/Python/PythonInterface.cpp

```
@@ -849,6 +849,56 @@ extern "C" {
+
+		geometry::SpatialGrid<FloatingPointType> space;
+		space.build(buildresult.atoms.points, cell, cutoff);
+		auto bonds = WITH_LOG_M(space, get_bonds, false);
+
+		// Flags intentionally correspond only to the asymmetric-unit inputs; 
+		// VoronoiDiagram resizes the flag vector and treats symmetry-expanded sites as false.
+		Diagram diag(buildresult.atoms.points, bonds, cell, bools);
+
+		auto ce = diag.extractCells();
+
+		cpplib::voronoi::VoronoiFused vf(ce, cell.fracToCart());
```

Cppcheck (2.20.0)

[style] The function 'getAngleRad' is never used.

(unusedFunction)